kafka-connect-aws-s3/src/it/scala/io/lenses/streamreactor/connect/aws/s3/sink/S3SinkTaskTest.scala

```diff
@@ -788,22 +779,40 @@ class S3SinkTaskTest extends AnyFlatSpec with Matchers with S3ProxyContainerTest
 
   }
 
-  "S3SinkTask" should "write to bytes format and combine files" in {
+  "S3SinkTask" should "writing to bytes format doesn't allow you to combine files" in {
+
+    val task = new S3SinkTask()
+
+    val props = DefaultProps
+      .combine(
+        Map(
+          "connect.s3.kcql" -> s"insert into $BucketName:$PrefixName select * from $TopicName STOREAS `BYTES` WITH_FLUSH_COUNT = 2",
+        ),
+      ).asJava
+
+    Try(task.start(props)) match {
+      case Failure(exception) =>
+        exception.getMessage should startWith("BYTES mode must be used in conjunction")
```

Comment:
Nitpick. Might be better a message saying the FlushCount > 1 is not allowed for BYTES. If you want to store N records as raw bytes use AVRO or PARQUET.